@xen-orchestra/web/src/components/hosts/HostsTable.vue

```diff
@@ -29,13 +29,16 @@
             @click="selectedHostId = row.id"
           >
             <td v-for="column of row.visibleColumns" :key="column.id" class="typo-body-regular-small">
-              <div v-if="column.id === 'host'">
+              <div v-if="column.id === 'host'" class="host">
                 <UiObjectLink :route="`/host/${row.id}/dashboard`" @click.stop>
                   <template #icon>
                     <VtsObjectIcon size="medium" :state="getHostState(column.value.state)" type="host" />
                   </template>
-                  {{ column.value.label }}
+                  <template #default>
```

Comment:
```suggestion
                  <template #default>
```
I don't think it's necessary to use this, but ok